src/api/services/studies.js

```diff
@@ -500,22 +500,33 @@ module.exports.addActivityToTest = async (options) => {
         return { status: 400, data: { message: 'You have not included yourself as owner of the activity' } };
       }
     }
-
+    console.log("Adding activity :" + JSON.stringify(test));
```

Comment:
beware of console.log. Use a propper logger (pino, winston, bunyan). Console log performance is bad (https://dev.to/adam_cyclones/consolelog-is-slow-2a2b) and removes the benefits of using nodejs

Reply:
fixed using pino